bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -378,7 +383,7 @@ object BagsManager {
       val k = ObsKey(o)
       val s = m.lookup(k).fold(IdleState(k, None): BagsState) {
         case IdleState(_, Some(hash)) => IdleState(k, hashObs(o).filter(_ === hash))
-        case RunningState(_, h)       => if (hashObs(o).contains(h)) RunningState(k, h) else RunningEditedState(k)
+        case RunningState(_, c, h)    => if (hashObs(o).contains(h)) RunningState(k, c, h) else RunningEditedState(k)
```

Comment:
I guess we could do two `case` statements instead, I would think.

I'm not sure it is clearer? I suspected myself that:
`case a if cond => x`
`case a => y`
would result in `y` if `cond` was `false`, but I wasn't 100% convinced until I checked to be sure.

I think maybe:
`case a => if cond x else y`
is clearer?